app/controllers/reviews_controller.rb

```diff
@@ -29,7 +31,7 @@ def find_booking
   end
 
   def review_params
-    params.require(:review).permit(:description, :rating, :user_id, :title, :booking_id, :job_id)
```

Comment:
Whats the reasoning in removing this? Each review instance shoudl have these values so the database get populated correctly.

Reply:
My thought process was that the only items you would receive from the review creation page were  description and rating. The other variables would be accessed directly on the page. This is why in the lines 12 to 13 in the create method, I have added the following code:
@review.user_id = @booking.user.id
@review.booking_id = @booking.id